### PR TITLE
Add compatibility check for WordPress 5.0 or Gutenberg.

### PR DESCRIPTION
**Pull request by kopepasah**
This pull request adds a compatibility check for WordPress 5.0 or Gutenberg plugin.

### CODE REVIEW

**Comment by kopepasah** [comment]
Will close #168

File: block-lab.php [hunk 1]
```diff
@@ -60,6 +60,48 @@ function block_lab_php_version_text() {
...
+
+/**
+ * String describing the minimum WP version or Gutenberg Plugin requirement.
+ *
+ * "Blocks" are a feature of WordPress 5.0+ or require the Gutenberg plugin.
+ *
+ * @return string
+ */
+function block_lab_wp_version_text() {
+	return __( 'Block Lab plugin error: Your version of WordPress is too old or Gutenberg Plugin is not installed. You must be running WordPress 5.0 or install the Gutenberg Plugin to use Block Labs.', 'block-lab' );
```
Comment (lukecarbis) [inline block-lab.php]: "… or _the_ Gutenberg Plugin is not installed"

Comment (lukecarbis) [reply block-lab.php]: Block Lab (not Block Labs)

**Review by lukecarbis — CHANGES_REQUESTED** [review]

File: block-lab.php (hunk 1 above)
Comment (kopepasah) [reply block-lab.php]: Fixed

**Review by lukecarbis — APPROVED** [review]

**Pull request merged**